### PR TITLE
Updated post onboarding (round 1)

### PR DESCRIPTION
**Pull request by broskoski**
Re: #634 

tips

This ignores the expanded tip with content for now and just sends the user to the explore page.

### CODE REVIEW

File: apps/user/components/tips/helpers.coffee [hunk 1]
```diff
@@ -0,0 +1,10 @@
+tips = require "./content.coffee"
+{ reject } = require 'underscore'
+
+
+module.exports = 
+  addTips: (user, author, cookies) ->
+    if (user?.id is author.id and user?.get('show_tour') isnt false)
```
Comment (dzucconi) [inline apps/user/components/tips/helpers.coffee]: `if (user?.id is author.id and not user?.get('show_tour'))`

Comment (broskoski) [inline apps/user/components/tips/helpers.coffee]: yes yes

File: apps/user/components/tips/index.styl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.Tips
+  @extends .flex-container
```
Comment (dzucconi) [inline apps/user/components/tips/index.styl]: Extract whatever is important essential about flex container into a mixin or inline it. This leads to very hard to refactor CSS unless you're extending something that's in the same file and closely conceptually related.

Comment (broskoski) [inline apps/user/components/tips/index.styl]: Yessir

File: apps/user/components/tips/view.coffee [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    e.preventDefault()
+    e.stopPropagation()
+
+    @setClosed()
+
+    if trigger = @$el.data('trigger')
+      switch trigger
+        when "new:channel"
+          # jQuery to open the new channel dialog, and close when body is clicked
+          $dropdown = $('.js-channel-create .js-dropdown')
```
Comment (dzucconi) [inline apps/user/components/tips/view.coffee]: Add a new `js-` selector to handle this. These on their own refer to very different actions.

Comment (broskoski) [inline apps/user/components/tips/view.coffee]: Yes sir

File: components/blocks/grid_item/templates/block.jade [hunk 4]
```diff
@@ -13,7 +13,5 @@ case block.get('class')
     include types/attachment
   when "Text"
     include types/text
-  when "Tip"
-    include types/tip
```
Comment (dzucconi) [inline components/blocks/grid_item/templates/block.jade]: Yay

**Comment by dzucconi** [comment]
Looks good aside from the things mentioned.

Am I the only one that finds those X's too chunky?

**Pull request merged**